Framework/Sources/UI/TweakViewController.swift

```diff
@@ -231,14 +231,16 @@ extension TweakViewController {
 extension TweakViewController: TweakViewControllerCellDelegate {
     
     internal func tweakConfigurationCellDidChangeValue(_ cell: TweakViewControllerCell) {
-        guard let featureValue = cell.feature, let variableValue = cell.variable ,
-              let indexPath = indexPathForTweak(with: featureValue, variable: variableValue),
-              let tweak = tweakAt(indexPath: indexPath)
-        else { return }
-        let feature = tweak.feature
-        let variable = tweak.variable
-        tweakManager.set(cell.value, feature: feature, variable: variable)
-        tweak.value = cell.value
+        if let feature = cell.feature, let variable = cell.variable {
```

Comment:
same here ([change in #80](https://github.com/justeat/JustTweak/pull/81/files#diff-e977a36c2345be54b8b1ada8ab760c787dcf0bc13dc9317669bd3c436512eb7cR234))